virttest/utils_time.py

```diff
@@ -0,0 +1,243 @@
+import logging
+import re
+
+from avocado.utils import process
+
+
+def get_host_timezone():
```

Comment:
`get_timezone(session=None)` can be introduced to get either host or guest based on session.

Reply:
Thanks for review. I updated `get_guest_clocksource` and `get_host_clocksource` to `get_clocksource` in below. Here I just want to get the timezone of host, the result will be used in  `sync_timezone()`.